Gordon360/Models/ViewModels/UserPrivacyUpdateViewModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
+namespace Gordon360.Models.ViewModels
+{
+    // Field: the field where the user wants to update the privacy setting (HomeCity, HomeState, HomeCountry, SpouseName, Country, HomePhone, MobilePhone)
```

Comment:
Let's add an overall comment about the purpose of this class.  So far, I see that it's used as an argument to UpdateUserPrivacyAsync in ProfileService.cs.